language/bytecode-verifier/transactional-tests/tests/reference_safety/move_one_branch.mvir

```diff
@@ -5,9 +5,10 @@ module 0x1.M {
         let x_ref: &mut u64;
         x = 0;
         x_ref = &mut x;
-        if (move(cond)) {
-            *move(x_ref) = 1;
-        }
+        if (move(cond)) b0 else b1;
+    label b0:
+        *move(x_ref) = 1;
+    label b1:
```

Comment:
This feels a bit weird that the cfg was flipped that the, but at least this one should be equivalent as b0 falls into b1